securedrop_client/gui/main.py

```diff
@@ -172,9 +171,15 @@ def on_source_changed(self):
         """
         source_item = self.main_view.source_list.currentItem()
         source_widget = self.main_view.source_list.itemWidget(source_item)
-        if source_widget:
-            self.current_source = source_widget.source
-            self.show_conversation_for(self.current_source, self.controller.is_authenticated)
+
+        # Show conversation for the currently-selected source if it hasn't been deleted. If the
+        # current source no longer exists, clear the conversation for that source.
+        # from PyQt5.QtCore import pyqtRemoveInputHook; pyqtRemoveInputHook()
+        # import pdb; pdb.set_trace()
```

Comment:
oops debug statements needs to get snipped out

Reply:
good catch!